src/types.ts

```diff
@@ -511,6 +511,138 @@ export type Mutation<R extends Record<string, Any> = Record<string, Any>> =
   | {delete: MutationSelection}
   | {patch: PatchMutationOperation}
 
+/** @public */
+export type Action =
+  | CreateAction
+  | ReplaceDraftAction
+  | EditAction
+  | DeleteAction
+  | DiscardAction
+  | PublishAction
+  | UnpublishAction
+
+/**
+ * Creates a new draft document. The published version of the document must not already exist.
+ * If the draft version of the document already exists the action will fail by default, but
+ * this can be adjusted to either leave the existing document in place or overwrite it.
+ */
+export type CreateAction = {
+  actionType: 'sanity.action.document.create'
+
+  /** * ID of the published document to create a draft for. */
+  publishedId: string
+
+  /** * Document to create. Requires a `_type` property. */
+  attributes: IdentifiedSanityDocumentStub
+
+  /** * ifExists controls what to do if the draft already exists */
+  ifExists: 'fail' | 'ignore'
+}
+
+/**
+ * Replaces an existing draft document.
+ * At least one of the draft or published versions of the document must exist.
+ */
+export type ReplaceDraftAction = {
+  actionType: 'sanity.action.document.replaceDraft'
+  /** * Draft document ID to replace, if it exists. */
```

Comment:
```suggestion

  /** * Draft document ID to replace, if it exists. */
```

Nit: formatting (missing empty line)